tsconfig.json

```diff
@@ -9,6 +9,7 @@
       "jsx": "react-native",
       "resolveJsonModule": true,
       "noImplicitAny": false,
+      "types": ["react", "react-native"]
```

Comment:
By default, TypeScript includes all globally installed type definitions it finds.

This way, we are only including type definitions from these two packages. Ignore everything else.

Reply:
We don't want "debug" in here too, since package.json specifies a dev dependency on "@types/debug"?